components/CampaignListItem/index.tsx

```diff
@@ -0,0 +1,121 @@
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined'
+import { Box } from '@mui/material'
+
+const styledmainbox = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '1070px',
+  height: '100%',
```

Comment:
the component looks quite big on my screen, is this expected? can help to check with xiaoen also if you are already checking with her on the design

Reply:
nice suggestion, I've changed this to 50% width instead of using fixed width, seems more responsive now, maybe because my monitor is bigger so it didn't look that huge on my end.